CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+# Main releases
```

Comment:
Why does this title appear twice in the file? I guess you wanted to move it, not copy.

Separately from that, why the null-safe version is set to 1.0.0? It looks like it's forked from a very old version of the library. I'd rather set it to 3.0.0-nullsafety.

Reply:
ah yess very good one I will change this